src/Mapbender/ManagerBundle/Resources/views/Repository/index.html.twig

```diff
@@ -29,13 +29,6 @@
             {% elseif source.valid is not null %}
                 <span class="iconWarningSmpl icon" title="{{"mb.manager.admin.source.notvalid"|trans}}"></span><br>
             {% endif %}
-            {% if source.status == 0 %}
```

Comment:
I think it's an fix, not new feature.
Please do it outside of this pull-request